src/Microsoft.Fx.Portability/ObjectModel/NuGetPackageInfo.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Runtime.Versioning;
+using System.Linq;
+
+namespace Microsoft.Fx.Portability.ObjectModel
+{
+    public class NuGetPackageInfo
+    {
+        public string AssemblyInfo { get; private set; }
+        public FrameworkName Target { get; private set; }
+        public IList<NuGetPackageId> SupportedPackages { get; private set; }
+
+        public NuGetPackageInfo( string assemblyInfo, FrameworkName target, IEnumerable<NuGetPackageId> supportedPackages)
```

Comment:
Please implement `Equals` and `GetHashCode`

Reply:
Done